app-admin/ngxtop/ngxtop-0.0.3_pre141201-r3.ebuild

```diff
@@ -1,11 +1,11 @@
-# Copyright 1999-2023 Gentoo Authors
+# Copyright 1999-2024 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=8
 
-PYTHON_COMPAT=( python3_{9..11} )
+PYTHON_COMPAT=( python3_{9..12} )
 PYTHON_REQ_USE="sqlite"
-
+DISTUTILS_USE_PEP517=setuptools
```

Comment:
Please keep it in the same block as `PYTHON*` stuff.

Also, while at it, could you check if we can remove `DEPEND` below? It seems unnecessary.

Reply:
It's needed for the tests, so added a condition to it.